src/ObjCRuntime/NativeString.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace ObjCRuntime {
+	internal struct NativeString : IDisposable {
```

Comment:
I wonder if we should call it `NativeUtf8String` instead, to avoid potential confusion with Objective-C strings (and maybe even Swift strings one day).

Reply:
It's not necessarily UTF8, so that would be misleading. What would you think about renaming it to `TransientString` or `TransientPlatformString` in order to stress that it's short lived?

Reply:
`TransientString` works for me!